algorithms/kernel/dtrees/forest/regression/oneapi/df_regression_train_dense_default_oneapi_impl.i

```diff
@@ -0,0 +1,1245 @@
+/* file: df_regression_train_dense_default_oneapi_impl.i */
```

Comment:
This file is too huge! Is it possible to divide it into several?

Reply:
hmm, all methods here are required for tree building. I can separate out functionality related to results processing (oob, varImprotance). What is the appropriate size?

Reply:
i am going to split it during classifier addition